.github/workflows/e2e_test_run.yaml

```diff
@@ -21,7 +21,7 @@ on:
 jobs:
   e2e-test:
     name: End-to-End Test Run
-    runs-on: [self-hosted, linux, x64, "${{ inputs.runner-tag }}"]
+    runs-on: [self-hosted, linux, "${{ inputs.runner-tag }}"]
```

Comment:
Should we modify existing test that runs on `x64` or add new tests for `arm64`?
I think we should test for both `x64` and `arm64`.
Applies for the other workflow as well.

Reply:
Do you mean we should run an e2e test for both x64 and arm64 at the same time? I think in this case the end-to-end test needs to be adapted to build an arm charm. I am afraid this is beyond the scope of this PR. 
I can revert to `x64` if you prefer, but I think removing `x64` is not harmful because the `runner-tag` is sufficient to identify the runner.

Reply:
It might be fine.